node-launchpad/src/app.rs

```diff
@@ -120,7 +120,7 @@ impl App {
         let change_connection_mode = ChangeConnectionModePopUp::new(connection_mode)?;
         let port_range = PortRangePopUp::new(connection_mode, port_from, port_to);
         let rewards_address = RewardsAddress::new(app_data.discord_username.clone());
-        let upgrade_nodes = UpgradeNodesPopUp::default();
+        let upgrade_nodes = UpgradeNodesPopUp::new(app_data.nodes_to_start);
```

Comment:
If I'm not wrong, this is set during the init of `UpgradeNodesPopUp`, what if user has added 2 more nodes? This would not be updated right?

Reply:
That is true. Also `fixed_inteval` maybe can be passed as an argument. I will review the first one.
Good catch!